src/api_client.py

```diff
@@ -5,66 +5,68 @@
-#print json.loads(jsonDeterministicAddresses)
-
-#print 'Uncomment this next line to print the first deterministic address that would be generated with the given passphrase. This will Not add it to the Bitmessage interface or the keys.dat file.'
-#print api.getDeterministicAddress('asdfasdfqwser'.encode('base64'),4,1)
-
-#print 'Uncomment this line to subscribe to an address. (You must use your own address, this one is invalid).'
-#print api.addSubscription('2D94G5d8yp237GGqAheoecBYpdehdT3dha','test sub'.encode('base64'))
-
-#print 'Uncomment this line to unsubscribe from an address.'
-#print api.deleteSubscription('2D94G5d8yp237GGqAheoecBYpdehdT3dha')
-
-print 'Let\'s now print all of our inbox messages:'
-print api.getAllInboxMessages()
-inboxMessages = json.loads(api.getAllInboxMessages())
-print inboxMessages
-
-print 'Uncomment this next line to decode the actual message data in the first message:'
-#print inboxMessages['inboxMessages'][0]['message'].decode('base64')
-
-print 'Uncomment this next line in the code to delete a message'
-#print api.trashMessage('584e5826947242a82cb883c8b39ac4a14959f14c228c0fbe6399f73e2cba5b59')
-
-print 'Uncomment these lines to send a message. The example addresses are invalid; you will have to put your own in.'
-#subject = 'subject!'.encode('base64')
-#message = 'Hello, this is the message'.encode('base64')
-#ackData = api.sendMessage('BM-Gtsm7PUabZecs3qTeXbNPmqx3xtHCSXF', 'BM-2DCutnUZG16WiW3mdAm66jJUSCUv88xLgS', subject,message)
-#print 'The ackData is:', ackData
-#while True:
-#    time.sleep(2)
-#    print 'Current status:', api.getStatus(ackData)
-
-print 'Uncomment these lines to send a broadcast. The example address is invalid; you will have to put your own in.'
-#subject = 'subject within broadcast'.encode('base64')
-#message = 'Hello, this is the message within a broadcast.'.encode('base64')
-#print api.sendBroadcast('BM-onf6V1RELPgeNN6xw9yhpAiNiRexSRD4e', subject,message)
+if __name__ == '__main__':
+
+    api = xmlrpclib.ServerProxy("http://bradley:password@localhost:8442/")
+
+    print 'Let\'s test the API first.'
+    inputstr1 = "hello"
+    inputstr2 = "world"
+    print api.helloWorld(inputstr1, inputstr2)
+    print api.add(2,3)
+
+    print 'Let\'s set the status bar message.'
+    print api.statusBar("new status bar message")
+
+    print 'Let\'s list our addresses:'
+    print api.listAddresses()
+
+    print 'Let\'s list our address again, but this time let\'s parse the json data into a Python data structure:'
+    jsonAddresses = json.loads(api.listAddresses())
+    print jsonAddresses
+    print 'Now that we have our address data in a nice Python data structure, let\'s look at the first address (index 0) and print its label:'
+    print jsonAddresses['addresses'][0]['label']
+
+    print 'Uncomment the next two lines to create a new random address with a slightly higher difficulty setting than normal.'
+    #addressLabel = 'new address label'.encode('base64')
```

Comment:
If this code not required, please clean it up.

Reply:
This work (see the PR description) is about adding docs and small fixes needed to support the docs parsing the project. There's other work going on to deal with style and lint. These are recognisable by their descriptions. Removing superfluous print statements is going a little beyond just style anyway since that affects the output of the program.

Reply:
Just realised you might be talking about addressLabel, which is indeed cruft and should be removed by the code_quality work.